### ISSUE TITLE
Update install instructions / deps? Install currently fails.

### ISSUE THREAD

**Issue by rsend** [barnacs/compy#73]
Hi,
I tried installing using the latest version of Go on a Raspberry Pi. The instructions now require "go install [url]@version", which fails at:

>  # github.com/chai2010/webp
> go/pkg/mod/github.com/chai2010/webp@v1.1.1/webp.go:22:9: undefined: webpGetInfo
> go/pkg/mod/github.com/chai2010/webp@v1.1.1/webp.go:26:20: undefined: webpDecodeGray
> go/pkg/mod/github.com/chai2010/webp@v1.1.1/webp.go:39:20: undefined: webpDecodeRGB
> go/pkg/mod/github.com/chai2010/webp@v1.1.1/webp.go:52:20: undefined: webpDecodeRGBA
> go/pkg/mod/github.com/chai2010/webp@v1.1.1/webp.go:68:14: undefined: webpDecodeGrayToSize
> go/pkg/mod/github.com/chai2010/webp@v1.1.1/webp.go:82:14: undefined: webpDecodeRGBToSize
> go/pkg/mod/github.com/chai2010/webp@v1.1.1/webp.go:96:14: undefined: webpDecodeRGBAToSize
> go/pkg/mod/github.com/chai2010/webp@v1.1.1/webp.go:109:7: undefined: toGrayImage
> go/pkg/mod/github.com/chai2010/webp@v1.1.1/webp.go:110:14: undefined: webpEncodeGray
> go/pkg/mod/github.com/chai2010/webp@v1.1.1/webp.go:119:14: undefined: webpEncodeRGB
> go/pkg/mod/github.com/chai2010/webp@v1.1.1/webp.go:119:14: too many errors
> github.com/pixiv/go-libjpeg/jpeg: build constraints exclude all Go files in /root/go/pkg/mod/github.com/pixiv/go-libjpeg@v0.0.0-20190822045933-3da21a74767d/jpeg
> gopkg.in/kothar/brotli-go.v0/dec: build constraints exclude all Go files in /root/go/pkg/mod/gopkg.in/kothar/brotli-go.v0@v0.0.0-20170728081549-771231d473d6/dec
> gopkg.in/kothar/brotli-go.v0/enc: build constraints exclude all Go files in /root/go/pkg/mod/gopkg.in/kothar/brotli-go.v0@v0.0.0-20170728081549-771231d473d6/enc

Note that I had to install libjpeg-9 instead of libjpeg-8 as the latter is now unavailable - would that cause the above issues? I'd love to use this project, but I can't tell if it's being maintained or not.

**Comment by gaul** [comment]
I suspect that you do not have libwebp and libwebp-dev installed.  Maybe the installation instructions are incomplete now that compy added webp support?

**Comment by rsend** [comment]
Hi,
So I did get it to work - but the workaround involves using libjpeg8 anyway. In particular for Raspberry Pi, I had to build [this](https://forums.raspberrypi.com/viewtopic.php?t=64725) which I guess provides libjpeg8-dev which seems to be required. That makes the errors in my first comment here go away.

So - do you think it's feasible to use libjpeg9? 

We also need a newer go version, which changes the install syntax from "go get" to "go install" as mentioned

**Comment by TehDomic** [comment]
> Hi, So I did get it to work - but the workaround involves using libjpeg8 anyway. In particular for Raspberry Pi, I had to build [this](https://forums.raspberrypi.com/viewtopic.php?t=64725) which I guess provides libjpeg8-dev which seems to be required. That makes the errors in my first comment here go away.
> 
> So - do you think it's feasible to use libjpeg9?
> 
> We also need a newer go version, which changes the install syntax from "go get" to "go install" as mentioned

Agreed. Would also like to use WebP or newer. I was manually able to figure out downloading the code in a zip file and build it in a Ubuntu VM with `golang-go` manually installed.
